autogen/agentchat/contrib/qdrant_retrieve_user_proxy_agent.py

```diff
@@ -304,10 +304,18 @@ class QueryResponse(BaseModel, extra="forbid"):  # type: ignore
         ),
     )
 
-    data = {
-        "ids": [[result.id for result in sublist] for sublist in results],
-        "documents": [[result.document for result in sublist] for sublist in results],
-        "distances": [[result.score for result in sublist] for sublist in results],
-        "metadatas": [[result.metadata for result in sublist] for sublist in results],
-    }
+    data = [
+        [
+            QueryResponse(
+                id=result.id,
+                embedding=result.embedding,
+                metadata=result.metadata,
+                document=result.document,
+                score=result.score,
+            )
+            for result in sublist
+        ]
+        for sublist in results
+    ]
+
```

Comment:
because the output is mentioned to be `List[List[QueryResponse]]` .

Reply:
This changed the data type. Please make sure if still works with the qdrant notebook.

Reply:
ill check on, if its working well with the notebook too.
@thinkall